app/components/Sidebar/components/SidebarLink.js

```diff
@@ -29,46 +26,25 @@ function SidebarLink({
   icon,
   children,
   onClick,
+  onMouseEnter,
   to,
   label,
   active,
   menu,
   menuOpen,
-  hideDisclosure,
   theme,
   exact,
   href,
+  innerRef,
   depth,
   ...rest
 }: Props) {
-  const [expanded, setExpanded] = React.useState(rest.expanded);
-
   const style = React.useMemo(() => {
     return {
       paddingLeft: `${(depth || 0) * 16 + 16}px`,
     };
   }, [depth]);
 
-  React.useEffect(() => {
-    if (rest.expanded !== undefined) {
-      setExpanded(rest.expanded);
-    }
-  }, [rest.expanded]);
-
-  const handleClick = React.useCallback(
-    (ev: SyntheticEvent<>) => {
-      ev.preventDefault();
-      ev.stopPropagation();
-      setExpanded(!expanded);
-    },
-    [expanded]
-  );
-
-  const handleExpand = React.useCallback(() => {
-    setExpanded(true);
-  }, []);
-
-  const showDisclosure = !!children && !hideDisclosure;
```

Comment:
🍖 all of this logic is moved into `DocumentLink`, which was the only place that was using it at all